src/core_ocean/shared/mpas_ocn_equation_of_state_jm.F

```diff
@@ -393,12 +393,14 @@ subroutine ocn_equation_of_state_jm_density_only(nVertLevels,      &
       end do
 #ifdef MPAS_OPENACC
       !$acc update host(density)
-      !$acc exit data delete(density, p, p2, tracerTemp, tracerSalt)
+      !$acc exit data delete(p, p2, tracerTemp, tracerSalt)
 #else
       !$omp end do
       !$omp end parallel
 #endif
 
+      call mpas_log_write('leaving eq of state')
+
```

Comment:
I don't recall adding this.  I'm not sure if it showed up during rebase?  Unless there is a need for this log statement, I would rather remove it...

Reply:
Part of our equation of state contact tracing ;^)    definitely remove